lib/puppetserver/ca/action/generate.rb

```diff
@@ -78,6 +78,10 @@ def self.parser(parsed = {})
             opts.on('--ttl TTL', 'The time-to-live for each cert generated and signed') do |ttl|
               parsed['ttl'] = ttl
             end
+            opts.on('--force', 'Suppress error when signing cert offline.',
+                    'To be used with ca-client') do |force|
```

Comment:
What do we want the behavior to be if `--force` is passed without `--ca-client`? The options I see are:
- just ignore the `--force` flag and proceed with the action
- log that `--force` is being ignored and proceed with the action
- fail fast and log that `--force` can't be used without `--ca-client`

Reply:
I was inclined to do the first, since it's not harmful, but am fine with either of the others too 🤷‍♀️

Reply:
Right now, it's the first option. The `--force` flag is not really use anywhere outside of the the `run` function. I can definitely add a check.